treescript/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.7
+
+RUN groupadd --gid 10001 app && \
+    useradd -g app --uid 10001 --shell /usr/sbin/nologin --create-home --home-dir /app app
+
+COPY . /app
```

Comment:
Hm, does this work? I'm guessing we're going to copy, e.g. `treescript/.` in as `/app/.` However, treescript requires `../scriptworker_client` (iscript requires that, plus `../vendored/mozbuild`) to install. We either need to also copy those directories in, or we need to populate the wheels somewhere.

Reply:
it doesn't work. for now (in this PR) i'm ignoring it since i'm only replacing that tox is running on taskcluster and not on travis.